shuup/xtheme/layout/_contact_group.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2018, Shuup Inc. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.utils.translation import ugettext_lazy as _
+
+from shuup.core.models import CompanyContact, PersonContact
+from shuup.xtheme.layout import Layout
+from shuup.xtheme.layout.utils import get_customer_from_context
+
+
+class AnonymousContactLayout(Layout):
+    identifier = "xtheme-anonymous-contact-layout"
+    help_text = _("Content in this placeholder is shown for guests.")
+
+    def is_valid_context(self, context):
+        customer = get_customer_from_context(context)
+        return (customer.is_anonymous if customer else False)
```

Comment:
is this test correct? Testing an `AnonymousContact` will always return `False`:
```py
>>> bool(AnonymousContact())
False
```
This way, the context should be valid if customer is `None` of `False` IMO